apps/api/src/user-account/repository.js

```diff
@@ -61,17 +71,98 @@ const getPaginatedList = async ({
         }));
 };
 
-const updateOne = ({ client, id, data }) => {
+const updateOne = async ({ client, id, data }) => {
```

Comment:
This became imbitable, can we split this?

Reply:
Mmm, not so easy because it's a chain of promises in the same transaction ....